Makefile

```diff
@@ -10,13 +10,13 @@ STACK_NAME=venafi-tests-stack
 #STACK_TEMPLATE=venafi/resources/tests/fixtures/test_certificate.yml
 STACK_TEMPLATE=venafi/resources/tests/fixtures/test_certificate_output_only.yml
 
-default: deploy test
+default: install test
 
-deploy:
-	python venafi/resources/tests/deploy_venafi_certificate_plugin.py
```

Comment:
According with Angel this file is not longer needed so it's suggested to delete it or mark it as deprecated to avoid mistakes using it.